src/CMakeLists.txt

```diff
@@ -326,19 +326,24 @@ if(NOT TARGET RapidJSON::RapidJSON)
 endif()
 
 option(WITH_FMT_HEADER_ONLY "use header-only version of fmt library" OFF)
-set(WITH_FMT_VERSION "8.1.1" CACHE
-  STRING "build with fmt version")
-find_package(fmt ${WITH_FMT_VERSION} QUIET)
+option(WITH_SYSTEM_FMT "build with system fmt" ON)
+if(WITH_SYSTEM_FMT)
+  find_package(fmt 9.1.0...<10.0.0)
```

Comment:
feel like this should include `REQUIRED` so cmake fails if system fmt is requested but not found

Reply:
Done, and defaulted `WITH_SYSTEM_FMT` to false to match.